src/qibolab/__init__.py

```diff
@@ -79,3 +79,39 @@ def execute_qasm(circuit: str, platform, runcard=None, initial_state=None, nshot
     return QibolabBackend(platform, runcard).execute_circuit(
         circuit, initial_state=initial_state, nshots=nshots
     )
+
+
+class MetaBackend:
+    """Meta-backend class which takes care of loading the qibolab backend."""
+
+    @staticmethod
+    def load(platform: str):
+        """Loads the backend.
+
+        Args:
+            platform (str): Name of the platform to load.
+        Returns:
+            qibo.backends.abstract.Backend: The loaded backend.
+        """
+        from qibolab.backends import QibolabBackend
+
+        PLATFORMS = os.environ.get(PLATFORMS)
+        if platform in PLATFORMS:
+            return QibolabBackend(platform=platform)
+        else:
+            raise_error(
+                ValueError,
+                f"Unsupported platform, please use one among {PLATFORMS}.",
+            )
+
+    def list_available(self) -> dict:
+        """Lists all the available qibolab platforms."""
+        available_backends = {}
+        for platform in os.environ.get(PLATFORMS):
+            try:
+                MetaBackend.load(platform=platform)
+                available = True
+            except:
+                available = False
+            available_backends[platform] = available
+        return available_backends
```

Comment:
```suggestion
        return {platform: True for platform in os.environ.get(PLATFORMS)}
```

Reply:
In the case of Qibolab, it does not make much sense to just load: you could have downloaded the driver, and not be able to connect anyhow.

Let's assume that if the platform is available, the backend is there (the other option would be trying to probe with a minimal execution, but it's not worth to execute on all machines just to list the backends)